cmd/envtool/main.go

```diff
@@ -216,14 +190,72 @@ func run(ctx context.Context, logger *zap.SugaredLogger) error {
 		}
 	}
 
+	return nil
+}
+
+// setupTigris configures Tigris.
+func setupTigris(ctx context.Context, logger *zap.SugaredLogger) error {
+	logger = logger.Named("tigris")
+
+	if err := waitForPort(ctx, logger, 8081); err != nil {
+		return err
+	}
+
+	cfg := &config.Driver{
+		URL: "127.0.0.1:8081",
+	}
+	driver, err := driver.NewDriver(ctx, cfg)
+	if err != nil {
+		return err
+	}
+	defer driver.Close()
+
+	logger.Info("Creating databases...")
+	for _, db := range []string{"admin", "test"} {
+		if err = driver.CreateDatabase(ctx, db); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// run runs all setup commands.
+func run(ctx context.Context, logger *zap.SugaredLogger) error {
+	go debug.RunHandler(ctx, "127.0.0.1:8089", logger.Named("debug").Desugar())
+
+	if err := setupPostgres(ctx, logger); err != nil {
+		return err
+	}
+
+	if err := setupTigris(ctx, logger); err != nil {
+		return err
+	}
```

Comment:
Are those commands fast? Should we run them in goroutines with waitgroup to make startup a bit faster?

Reply:
The waitgroup was there before, with a bug :-) https://github.com/FerretDB/FerretDB/pull/788#pullrequestreview-1017562742

Right now we do very little in those functions, they are fast

Reply:
😅